Fixes problem with the mismatch of binary names and 
man page descriptions of openldap client tools. 
Patch was developed in-house; it is Solaris specific and
will not be contributed upstream.

--- openldap-2.4.44/doc/man/man1/ldapcompare.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapcompare.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 1998-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldapcompare \- LDAP compare tool
+/usr/bin/ldapcompare \- LDAP compare tool
+.br
+/usr/bin/openldapcompare \- LDAP compare tool
+.br
+/usr/lib/openldap/bin/ldapcompare \- LDAP compare tool
 .SH SYNOPSIS
 .B ldapcompare
 [\c

--- openldap-2.4.44/doc/man/man1/ldapdelete.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapdelete.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 1998-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldapdelete \- LDAP delete entry tool
+/usr/bin/ldapdelete \- LDAP delete entry tool
+.br
+/usr/bin/openldapdelete \- LDAP delete entry tool
+.br
+/usr/lib/openldap/bin/ldapdelete \- LDAP delete entry tool
 .SH SYNOPSIS
 .B ldapdelete
 [\c

--- openldap-2.4.44/doc/man/man1/ldapexop.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapexop.1	Tue Dec  1 15:28:25 2015
@@ -15,7 +15,11 @@
 .TH LDAPEXOP 1
 
 .SH NAME
-ldapexop \- issue LDAP extended operations
+/usr/bin/ldapexop \- issue LDAP extended operations
+.br
+/usr/bin/openldapexop \- issue LDAP extended operations
+.br
+/usr/lib/openldap/bin/ldapexop \- issue LDAP extended operations
 
 .SH SYNOPSIS
 ldapexop

--- openldap-2.4.44/doc/man/man1/ldapmodify.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapmodify.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 1998-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldapmodify, ldapadd \- LDAP modify entry and LDAP add entry tools
+/usr/bin/ldapmodify, /usr/bin/ldapadd \- LDAP modify entry and LDAP add entry tools
+.br
+/usr/bin/openldapmodify, /usr/bin/openldapadd \- LDAP modify entry and LDAP add entry tools
+.br
+/usr/lib/openldap/bin/ldapmodify, /usr/lib/openldap/bin/ldapadd \- LDAP modify entry and LDAP add entry tools
 .SH SYNOPSIS
 .B ldapmodify
 [\c

--- openldap-2.4.44/doc/man/man1/ldapmodrdn.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapmodrdn.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 1998-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldapmodrdn \- LDAP rename entry tool
+/usr/bin/ldapmodrdn \- LDAP rename entry tool
+.br
+/usr/bin/openldapmodrdn \- LDAP rename entry tool
+.br
+/usr/lib/openldap/bin/ldapmodrdn \- LDAP rename entry tool
 .SH SYNOPSIS
 .B ldapmodrdn
 [\c

--- openldap-2.4.44/doc/man/man1/ldappasswd.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldappasswd.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 1998-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldappasswd \- change the password of an LDAP entry
+/usr/bin/ldappasswd \- change the password of an LDAP entry
+.br
+/usr/bin/openldappasswd \- change the password of an LDAP entry
+.br
+/usr/lib/openldap/bin/ldappasswd \- change the password of an LDAP entry
 .SH SYNOPSIS
 .B ldappasswd
 [\c

--- openldap-2.4.44/doc/man/man1/ldapsearch.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapsearch.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 1998-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldapsearch \- LDAP search tool
+/usr/bin/ldapsearch \- LDAP search tool
+.br
+/usr/bin/openldapsearch \- LDAP search tool
+.br
+/usr/lib/openldap/bin/ldapsearch \- LDAP search tool
 .SH SYNOPSIS
 .B ldapsearch
 [\c

--- openldap-2.4.44/doc/man/man1/ldapurl.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapurl.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 2008-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldapurl \- LDAP URL formatting tool
+/usr/bin/ldapurl \- LDAP URL formatting tool
+.br
+/usr/bin/openldapurl \- LDAP URL formatting tool
+.br
+/usr/lib/openldap/bin/ldapurl \- LDAP URL formatting tool
 .SH SYNOPSIS
 .B ldapurl
 [\c

--- openldap-2.4.44/doc/man/man1/ldapwhoami.1.old	Mon Nov 30 09:02:00 2015
+++ openldap-2.4.44/doc/man/man1/ldapwhoami.1	Tue Dec  1 15:28:25 2015
@@ -3,7 +3,11 @@
 .\" Copyright 1998-2016 The OpenLDAP Foundation All Rights Reserved.
 .\" Copying restrictions apply.  See COPYRIGHT/LICENSE.
 .SH NAME
-ldapwhoami \- LDAP who am i? tool
+/usr/bin/ldapwhoami \- LDAP who am i? tool
+.br
+/usr/bin/openldapwhoami \- LDAP who am i? tool
+.br
+/usr/lib/openldap/bin/ldapwhoami \- LDAP who am i? tool
 .SH SYNOPSIS
 .B ldapwhoami
 [\c
